src/sentry/api/urls.py

```diff
@@ -796,9 +796,9 @@
                     name="sentry-api-0-organization-key-transactions",
                 ),
                 url(
-                    r"^(?P<organization_slug>[^\/]+)/key-transactions-count/$",
-                    KeyTransactionCountEndpoint.as_view(),
-                    name="sentry-api-0-organization-key-transactions-count",
+                    r"^(?P<organization_slug>[^\/]+)/key-transactions-list/$",
+                    KeyTransactionListEndpoint.as_view(),
```

Comment:
I'd prefer to explicitly call out `list` here because the other endpoint is named `KeyTransactionEndpoint` and if the difference is just a single `s`, that will be very confusing to read.